mprof

```diff
@@ -405,6 +407,15 @@ def plot_file(filename, index=0, timestamps=True):
 
 
 def plot_action():
+    def get_comma_separated_args(option, opt, value, parser):
+        try:
+            newvalue = [float(x) for x in value.split(',')]
+        except:
+            raise OptionValueError("'%s' option must contain two numbers separated with a comma" % value)
+        if len(newvalue) != 2:
```

Comment:
Yes, that is taking a bit longer to do, as while `None` works, it then doesn't work for limiting which brackets are plotted... I have something working locally, but not happy with the code.